src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -2397,6 +2397,8 @@ def set_output_embeddings(self, new_embeddings):
         checkpoint=_CHECKPOINT_FOR_DOC,
         output_type=MaskedLMOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_output="'here'",
```

Comment:
Sorry to comment so late here. Could we maybe overwrite the BigBird example as well? https://huggingface.co/google/bigbird-roberta-base has quite a significant number of downloads and it's know to be a **long-range** model. Could we maybe provide a long input to be masked here and to all other examples below as well? 

Would be great if we could overwrite the example doc string here @ydshieh

Reply:
I think it's a great idea. Let me prepare better examples for Bigbird.